src/Components/WeatherOnMarsImage/WeatherOnMarsImage.scss

```diff
@@ -28,6 +28,14 @@
 
 }
 
+@media(min-width: 414px){
+
+  .iframeContainer iframe{
+    height: 73%
+  }
```

Comment:
These numbers look scarily specific and very fragile! (though possibly fine for now! :) )

If you had a little more time, is there a cleaner way you could think of to make this work?

Reply:
I agree it's a horrible fix, I would need to look into more detail how iframes work but I don't have the time unfortunately!